src/meta/src/controller/catalog/mod.rs

```diff
@@ -1092,6 +1089,26 @@ impl CatalogControllerInner {
         }
     }
 
+    pub(crate) fn notify_cancelled(&mut self, database_id: Option<DatabaseId>, id: ObjectId) {
```

Comment:
Do we have any call with `database_id` as `None`? If not, we can change to always pass `database_id: DatabaseId`.